### DISCUSSION TITLE
Guidance (do/don't) block

### DISCUSSION THREAD

**Discussion by embostan** [storybookjs/storybook#24888] [Ideas]
### Is your feature request related to a problem? Please describe.

It'd be great to have an MDX block for do's/dont's (similar to what you see in the Material Design documentation for instance)

### Describe the solution you'd like

Such a syntax for instance

```mdx
<Guidance>
    <Do>
        My text and maybe some image, story...
    </Do>
    <Dont>
        My text and maybe some image, story...
    </Dont>
</Guidance>
``` 

### Describe alternatives you've considered

Using basic MDX paragraphs, but it doesnt look as nice

### Are you able to assist to bring the feature to reality?

no

**Comment by valentinpalkovic** [comment]
Makes a lot of sense to me. WDYT @JReinhold and @cdedreuille?

**Comment by JReinhold** [comment]
This is a good idea, but it also sounds like something that isn't dependent on Storybook internals, those types of components are easy to build in userland (unless I'm missing something).

We tend to provide primitive Doc Blocks for interacting with Storybook core concepts, and any other components that are "just" nice UI should be built by users or made available in community packages.
